<launch>

    <node pkg="tf" type="static_transform_publisher" name="my_frame_broadcaster" args="0 0 0 0 0 0 1 map my_frame 10" />

         <!-- Show in Rviz    -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find planar_monocular_slam_thesis)/rviz/planar_monocular_slam_config.rviz" required="true"  output="screen" launch-prefix="xterm -e" />
    
    <node pkg="rosbag" type="play" name="player" 
          args="/home/paolo/april_tag_corridor_rectified_2021-05-26-14-19-27.compressed.bag /home/paolo/2022-06-29-11-27-08.bag"/>

    <node pkg="rosbag" type="play" name="player2" 
          args="/home/paolo/2022-06-29-11-27-08.bag"/>

    <node name = "planar_monocular_slam_node" args="_param:=condensed" pkg="planar_monocular_slam_thesis"  type="planar_monocular_slam_node" output="screen" >
        <remap from="/camera/image_raw" to="/camera/fisheye1_rect/image_raw"/>
        <remap from="/camera_info" to="/camera/fisheye1_rect/camera_info"/>
    </node>


</launch>
